### PR TITLE
feat(prospectType): [INFRA-1081] Update prospect types

### PR DESCRIPTION
**Pull request by Herraj**
## Goal
Updating prospect types and the scheduled surfaces with the new prospect types to reflect the changes in this [PR](https://github.com/Pocket/prospect-api).

JIRA ticket:
* [INFRA-1081]

[INFRA-1081]: https://getpocket.atlassian.net/browse/INFRA-1081?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sri-kirsh — APPROVED** [review]

File: src/shared/types.ts [hunk 1]
```diff
@@ -60,9 +63,11 @@ export const ScheduledSurfaces: ScheduledSurface[] = [
     guid: 'NEW_TAB_DE_DE',
     ianaTimezone: 'Europe/Berlin',
     prospectTypes: [
-      ProspectType.GLOBAL,
-      ProspectType.ORGANIC_TIMESPENT,
+      ProspectType.COUNTS,
```
Comment (sri-kirsh) [inline src/shared/types.ts:66]: could you provide lil bit context on why don't have recommend for berlin and US, but for Europe and Asia.

File: src/shared/types.ts [hunk 2]
```diff
@@ -71,9 +76,10 @@ export const ScheduledSurfaces: ScheduledSurface[] = [
     guid: 'NEW_TAB_EN_GB',
     ianaTimezone: 'Europe/London',
     prospectTypes: [
-      ProspectType.GLOBAL,
-      ProspectType.ORGANIC_TIMESPENT,
-      ProspectType.APPROVED,
+      ProspectType.COUNTS,
```
Comment (sri-kirsh) [inline src/shared/types.ts:79]: same way the acceptance criteria mentions we need to have `TitleUrlLearning` for all surfaces. is this same as `TITLE_URL_MODELED`?

Comment (sri-kirsh) [reply src/shared/types.ts:79]: asking coz other surfaces have `TITLE_URL_MODELED` except this

Comment (sri-kirsh) [reply src/shared/types.ts:79]: seems like  a DP requirement, following up these questions in this doc: https://getpocket.atlassian.net/wiki/spaces/PE/pages/2564587582/Prospecting+Sources#New-Tab-en-GB%3A--NEW_TAB_EN_GB

Looks good as per the docs.

**Review by sri-kirsh — APPROVED** [review]

File: src/shared/types.ts [hunk 3]
```diff
@@ -103,9 +109,11 @@ export const ScheduledSurfaces: ScheduledSurface[] = [
     guid: 'NEW_TAB_EN_INTL',
     ianaTimezone: 'Asia/Kolkata',
     prospectTypes: [
-      ProspectType.GLOBAL,
-      ProspectType.ORGANIC_TIMESPENT,
-      ProspectType.APPROVED,
+      ProspectType.COUNTS,
+      ProspectType.TIMESPENT,
+      ProspectType.RECOMMENDED,
+      ProspectType.TITLE_URL_MODELED,
```
Comment (sri-kirsh) [inline src/shared/types.ts:115]: url modeled is not included in this doc: 
Screen Shot 2023-05-02 at 10 59 38 AM
should we be removing this?

Comment (Herraj) [reply src/shared/types.ts:115]: yep, incorrectly added. Will remove, thanks!

File: src/shared/types.ts [hunk 4]
```diff
@@ -43,15 +44,17 @@ export const ScheduledSurfaces: ScheduledSurface[] = [
...
     ianaTimezone: 'America/New_York',
     prospectTypes: [
+      ProspectType.COUNTS,
+      ProspectType.TIMESPENT,
+      ProspectType.RECOMMENDED,
       ProspectType.TOP_SAVED,
-      ProspectType.GLOBAL,
-      ProspectType.ORGANIC_TIMESPENT,
-      ProspectType.SYNDICATED_NEW,
+      ProspectType.DOMAIN_ALLOWLIST,
```
Comment (sri-kirsh) [inline src/shared/types.ts:51]: is domain-allow list same as domain-allowlist new tab en-us? 
Screen Shot 2023-05-02 at 10 59 38 AM

File: src/shared/types.ts (hunk 1 above)
Comment (sri-kirsh) [reply src/shared/types.ts:66]: will follow-up in doc.

File: src/shared/types.ts (hunk 4 above)
Comment (sri-kirsh) [reply src/shared/types.ts:51]: redirected to doc questions

Comment (Herraj) [reply src/shared/types.ts:51]: Not sure about your question but we have one `DOMAIN_ALLOWLIST` type which is being used by new_tab_en_us, new_tab_de_de, new_tab_es_es, new_tab_fr_fr and new_tab_it_it :thinking:

**Review by sri-kirsh — APPROVED** [review]

**Pull request merged**